bucket/arduino.json

```diff
@@ -1,11 +1,11 @@
 {
-    "version": "1.8.19",
+    "version": "2.0.0",
     "description": "Arduino IDE",
     "homepage": "https://www.arduino.cc",
     "license": "GPL-2.0-or-later",
-    "url": "https://downloads.arduino.cc/arduino-1.8.19-windows.zip",
-    "hash": "sha512:c4072d808aea3848bceff5772f9d1e56a0fde02366b5aa523d10975c54eee2ca8def25ee466abbc88995aa323d475065ad8eb30bf35a2aaf07f9473f9168e2da",
-    "extract_dir": "arduino-1.8.19",
+    "url": "https://downloads.arduino.cc/arduino-ide/arduino-ide_2.0.0_Windows_64bit.zip",
+    "hash": "sha512:bd1b129030abc6185e69057cb3677448854e3cbdc5e984abcc093d3e1add84474794380ad03c417c20bafd265ba9f7567fde51874151f00af4593da231df6a81",
+    "extract_dir": "arduino-2.0.0",
```

Comment:
Remove extract_dir as this directory no longer exists in version 2.0.0. Please also remove it from autoupdate.